doc/aws.md

```diff
@@ -107,8 +97,8 @@ This template deploys the application servers, the load balancer, and also a sma
   - LogGroupPrefix: Leave this at the default: `/yet/lrsql/`
   - LogGroupRetentionInDays: Leave this at the default of 7 (days)
   - LrsVersion: Select the desired version of SQL LRS from the GitHub Releases page [here](https://github.com/yetanalytics/lrsql/releases)
-  - S3Bucket: The bucket created or used in Step 1
-  - S3Key: The zip filename from Step 1
+  - S3Bucket: Leave this at the default: `lrsql-dbfn`
+  - S3Key: Leave this at the default: (blank)
```

Comment:
Are you not going to mention that it's going to take the dbfn from our S3 bucket?

Reply:
Implementation detail, didn't think it relevant to the tutorial

Reply:
I dunno, I think it's important to show that resources like the dbfn aren't being sprung from thin air.

Reply:
> sprung from thin air

It's the cloud lol

But seriously we don't do an in-depth rundown of the other template components/resources, why would we do that for this specifically? If they are curious the templates are right there and decently description-ed.

Reply:
One change I _do_ want to make here tho, specify that the S3Key is an override (like prepend `Override` to it)
This will have the added advantage of letting folks update w/o having to clear the param out

Reply:
https://github.com/yetanalytics/lrsql/pull/279/commits/dd7569914ecfffacb031cd78a31522d5b9ede483